File: 1 - BEGINNER/C/1009 - Salário com Bônus.c
#include <stdio.h>
 
int main() {
 
    char nome;
    double salario_fixo, total_vendas, salario_total;
    
    scanf("%s", &nome);
    scanf("%lf", &salario_fixo);
    scanf("%lf", &total_vendas);
    
    /*
    Salário total do funcionário, deve ser seu salário fixo, somado a 15% do que ele vendeu no mês, ou seja, 0.15 * total_vendas é seu bônus
    */
   
    salario_total = salario_fixo + (total_vendas * 0.15);
    
    printf("TOTAL = R$ %.2lf\n", salario_total);
    
    return 0;
}

File: 1 - BEGINNER/Java/1009 - Salário com Bônus.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        String nome = input.nextLine();
        double salario_fixo = input.nextDouble();
        double vendas = input.nextDouble();
        double salario_total = vendas * 15.0 / 100.0 + salario_fixo;
        
        System.out.printf("TOTAL = R$ %.2f\n", salario_total);

        input.close();
 
    }
 
}